### ISSUE TITLE
Addon does not work with ember-cli 3.0.0(-beta.2)

### ISSUE THREAD

**Issue by jacobq** [mansona/ember-cli-notifications#200]
I have created a sample application [here](https://github.com/jacobq/ecn-with-ecli-3.x) that demonstrates the problem. The error looks like this in this case:

```
Uncaught Error: Could not find module `ember-cli-notifications/styles/components/notification-container` imported from `ember-cli-notifications/components/notification-container`
```

It seems to be related to [this line](https://github.com/stonecircle/ember-cli-notifications/blob/master/addon/components/notification-container.js#L5):
```javascript
import styles from '../styles/components/notification-container';
```
If I comment that line out and replace it with `const styles = {}` or create the file `../styles/components/notification-container.js` this error goes away.

**Comment by jacobq** [comment]
This is an upstream problem and has now been fixed. We just need to upgrade to the new version in ember-cli-notification's `package.json`. (See https://github.com/salsify/ember-css-modules/pull/95 ) I will create a PR for this shortly (if greenkeeper doesn't beat me to it).

**Comment by ahopkins** [comment]
Is t his working on Ember 3 yet? I was getting some errors when I tried upgrading. What version of this package do I need to be using?

**Comment by lamabiker** [comment]
@ahopkins not yet, but @jacobq 's PR should fix it #201

**Comment by mansona** [comment]
This is now fixed in https://github.com/stonecircle/ember-cli-notifications/pull/197

Let me know if it is all working with your apps 👍
